taipy/gui/types.py

```diff
@@ -82,6 +82,10 @@ class PropertyType(Enum):
     """
     The property holds a dictionary.
     """
+    dynamic_date = "dynamicdate"
+    """
+    The property holds a dynamic date.
```

Comment:
The proper wording should be: "The property is dynamic and holds a  date."
Can you please change the surrounding property types as well?

Reply:
I've changed based on your suggestion